backend/api/v1/endpoints/connections.py

```
@@ -56,13 +56,38 @@ def list_connections(db: Session = Depends(get_db), current_user: User = Depends
+        # Fetch users who have sent a request to the current user
+        received_requests = db.query(Connection.user_id).filter(
+            Connection.friend_id == current_user.id, 
+            Connection.status == "pending"
+        )
+
+        # Get users who are not in any of the above lists
+        users = db.query(User).filter(
+            User.id.notin_(friends),
+            User.id.notin_(sent_requests),
+            User.id.notin_(received_requests),
+            User.id != current_user.id  # Exclude self
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)